docs/fundamentals/documentarray/access-elements.md

```diff
@@ -217,11 +218,49 @@ Last but not the least, you can use integer, or integer slice to restrict the se
 This can be useful when you want to get top matches of all matches from all Documents, e.g.:
 
 ```python
-da['@m:5']
+da['@m[:5]']
 ```
 
 You can add space in the path-string for a better readability.
 
+## Index by multi-modal attributes
+In case you define a multi-modal Document schema and you create a DocumentArray of multi-modal Documents, you can 
+select attributes inside each Document:
+
+```python
+from docarray import Document, DocumentArray
+from docarray.dataclasses.types import dataclass, Text, Image
+
+
+@dataclass
+class MultiModalDocument:
+    text: Text
+    image: Image
+
+
+da = DocumentArray(
+    [
+        Document(MultiModalDocument(text='cat', image='cat.jpg')),
+        Document(MultiModalDocument(text='dog', image='dog.jpg')),
+        Document(MultiModalDocument(text='bird', image='bird.jpg')),
+    ]
+)
+
+# plot images
+da['@r.[image]'].plot_image_sprites()
+
+# print text attributes
+for text_doc in da['@r.[text]']:
+    print(text_doc.text)
+
+# select both attributes
+da['@r.[image, text]']
+
+# select attributes of the Documents at a given postion
+da['@r[1].[text]']
+```
+
+
```

Comment:
please read the new docs before continuing on this PR, this section needs a rewrite and not needed at all

Reply:
Should I completely remove the section (and keep the other minor changes) or remove the content and point to the selection documented in the dataclass section ?